release.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Is there a reason not to have this in `/scripts`?

Reply:
Yes! I plan to eliminate `/scripts` when the Docker infra works on macOS.